src/domain/tagset/tagset.resolver.ts

```diff
@@ -17,35 +15,35 @@ export class TagsetResolver {
     RestrictedGroupNames.EcoverseAdmins
   )
   @UseGuards(GqlAuthGuard)
-  @Mutation(() => Tagset, {
+  @Mutation(() => Boolean, {
```

Comment:
The result of a mutation must return object that contains a primary key. It is crucial to have the Identifier in the result, because the ApolloClient cache is using it to updates its internal cache.

Reply:
ah, was not aware of this...